docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+---
+services:
+  workflow.pacta.webapp:
+    build: .
+    # stdin_open: true
+    # tty: true
+    # entrypoint: ["R", "--args"]
+    # command: '/workflow.pacta.webapp/tests/params/full_23q4.json'
+    environment:
+      LOG_LEVEL: TRACE
+      ANALYSIS_OUTPUT_DIR: "/mnt/analysis_output_dir"
+      BENCHMARKS_DIR: "/mnt/benchmarks_dir"
+      REPORT_OUTPUT_DIR: "/mnt/report_output_dir"
+      SUMMARY_OUTPUT_DIR: "/mnt/summary_output_dir"
+      REAL_ESTATE_DIR: "/mnt/real_estate_dir"
+      SURVEY_DIR: "/mnt/survey_dir"
+      PACTA_DATA_DIR: "/mnt/pacta-data"
+      OUTPUT_DIR: "/mnt/analysis_output_dir"
+      PORTFOLIO_DIR: "/mnt/portfolios"
+    volumes:
+      - type: bind
+        source: ${data_dir:-./pacta-data/}
+        target: /mnt/pacta-data
+        read_only: true
+      - type: bind
+        source: ${input_dir:-./tests/portfolios/}
+        target: /mnt/portfolios
+        read_only: true
+      - type: bind
+        source: ${benchmarks_dir:-./benchmarks_dir/}
+        target: /mnt/benchmarks_dir/
+        read_only: true
+      - type: bind
+        source: ${analysis_output_dir:-./analysis_output_dir/}
+        target: /mnt/analysis_output_dir
+        read_only: false
+      - type: bind
+        source: ${real_estate_dir:-./real_estate_dir/}
+        target: /mnt/real_estate_dir
+        read_only: true
```

Comment:
Pretty sure this is safe to remove? Real estate has been wiped from PACTA (e.g. https://github.com/RMI-PACTA/workflow.transition.monitor/pull/308)

We're probably going to need to manually review all the latest commits in `workflow.transition.monitor` and `workflow.pacta`/`workflow.pacta.report` to make sure the have all the same recent updates...

Reply:
@AlexAxthelm do you mind opening a new issue(s) to track that?

Reply:
#6

Reply:
Thanks!